frontend/src/app/supportAdmin/DeviceType/DeviceForm.tsx

```diff
@@ -105,6 +107,16 @@ const DeviceForm = (props: Props) => {
             convertedSupportedDisease["testOrderedLoincCode"] =
               supportedDisease.testOrderedLoincCode;
           }
+          if (supportedDisease.testOrderedLoincLongName) {
```

Comment:
non-blocking for followup: I think if we type `convertedSupportedDisease` on line 86 as `SupportedDiseasesFormData`, we can get rid of these `@ts-ignores` since those fields are already marked optional in the type declaration

Reply:
Ohh! Nice catch! Done 😸